modules/programs/less.nix

```
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.less;
+in {
+  meta.maintainers = [ maintainers.pamplemousse ];
+
+  options = {
+    programs.less = {
+      extraConfig = mkOption {
```

I'm not certain `extraConfig` is the best here since it's not clear what it actually configures, especially considering the many ways you can affect the behavior of less. Since this goes in the lesskeys file I would suggest simply naming it
```suggestion
      keys = mkOption {
```